doc/faq.rst

```diff
@@ -26,3 +26,23 @@ The message means that there is more data stored in the repo than
 strictly necessary. With high probability this is duplicate data. In
 order to clean it up, the command ``restic prune`` can be used. The
 cause of this bug is not yet known.
+
+How can I specify encryption passwords automatically?
+-----------------------------------------------------
+
+When you run ``restict create``, you need to enter the passphrase on
```

Comment:
Here's a typo: `s/restict/restic/`, and I suspect you mean `restic backup`?

Reply:
oops, confused with borg, sorry, will reword.

Reply:
done.